perun-core/src/main/resources/perun-core.xml

```diff
@@ -495,6 +496,11 @@ http://www.springframework.org/schema/context http://www.springframework.org/sch
 			<property name="auditer" ref="auditer"/>
 			<property name="cacheManager" ref="cacheManager"/>
 		</bean>
+
+		<!-- PerunRolesLoader implementation -->
+		<bean id="perunRolesLoader" class="cz.metacentrum.perun.core.impl.PerunRolesLoader">
+			<property name="configurationPath" value="file:@perun.conf@perun-roles.yml"/>
```

Comment:
Probably not necessary now, but it would be nice, if it could fallback also to ``file:${perun.conf.custom}perun-roles.yml``, since we fill default path on build and later is for runtime change by settings ENV variable ``perun.conf.custom`` for tomcat.

Also, we can probably use this principle to switch fixed path of ``/etc/perun/perun-extSources.xml`` to build/runtime relative in ExtSourcesManager.